io.github.anirbandey1.ChatGPTElectron.yml

```diff
@@ -0,0 +1,63 @@
+app-id: io.github.anirbandey1.ChatGPTElectron
+runtime: org.freedesktop.Platform
+runtime-version: '22.08'
+sdk: org.freedesktop.Sdk
+base: org.electronjs.Electron2.BaseApp
+base-version: "22.08"
+command: run.sh
+rename-desktop-file: chatgpt-electron.desktop
+rename-icon: chatgpt-electron
+finish-args:
+  - --share=ipc
+  - --share=network
+  - --socket=x11
+  - --socket=pulseaudio
+  - --device=dri
+  # - --filesystem=home
```

Comment:
```suggestion
```

Reply:
Filesystem permission is not need for this project. So, I will just remove this comment

Reply:
fixed in [commit d946ebe2313b72722ff965164699292c8a806899 ](https://github.com/flathub/flathub/pull/4305/commits/d946ebe2313b72722ff965164699292c8a806899)